package/MDAnalysis/coordinates/LAMMPS.py

```diff
@@ -66,22 +66,36 @@
 :class:`~MDAnalysis.topology.LAMMPSParser.DATAParser`) together with a
 LAMMPS DCD with "*real*" provide the keyword *format="LAMMPS*"::
 
-    >>> u = MDAnalysis.Universe("lammps.data", "lammps_real.dcd", format="LAMMPS")
+..testsetup::
```

Comment:
This code block should be placed right after the title level **Example: Loading xxx**